package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "private": true,
+  "scripts": {
+    "test": "npm run with-server 'npm-run-all test:*'",
+    "test:rust": "cargo test",
+    "test:ruby": "cd ruby-sdk && bundle exec rspec",
+    "with-server": "start-server-and-test start-mock-server http://127.0.0.1:8378",
+    "start-mock-server": "npm start --prefix ./mock-server"
```

Comment:
This is going to be so great in the monorepo

Reply:
Yeah, I thought it would be awful to re-invent http mocking in every language